internal/indexer/postgres/bootstrap.sql

```diff
@@ -60,9 +60,10 @@ CREATE TABLE package (
     id SERIAL PRIMARY KEY,
     name text NOT NULL,
     kind text NOT NULL,
-    version text NOT NULL
+    version text NOT NULL,
+    module text,
```

Comment:
@Allda this causes an little bit of an issue.

We are trying to keep the package table de duplicated and small. In order to do this we use a unique constraint. However since you are not requiring module to "not be null" this will allow duplicates in the database for records such as

{ "package-a", "binary", "v0.0.1", null }

With the schema as is, I could enter that many times and take up considerable amount of space over time.

What we've done to avoid this is set it to NOT NULL and our code inputs the zero value "" empty string. In that case inserting

{ "package-a", "binary", "v0.0.1", "" } <- if module NOT NULL, this is only inserted ONCE.

Reply:
I added NOT NULL and use "" as a default which prevents from inserting duplicated items.